### PR TITLE
add colors and border radius to tailwind config

### PR DESCRIPTION
**Pull request by auqibyousuf**

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -14,6 +15,22 @@ const config: Config = {
...
+        grey: "#D7D7D7",
+        lightGrey: "E4E4E4",
+      },
+    },
+    borderRadius: {
+      sm: " 5px",
+      md: "8px",
+      lg: "16px",
+      xl: "30px",
+      full: "100%",
```
Comment (UmmarHamid) [inline tailwind.config.ts]: WE can name these as 5,8, ...
So that we don't need to remember what values sm has etc.
But again that is personal choice

**Review by UmmarHamid — APPROVED** [review]
LGTM!

File: tailwind.config.ts (hunk 1 above)
Comment (auqibyousuf) [reply tailwind.config.ts]: ok changing

Comment (auqibyousuf) [reply tailwind.config.ts]: Done

**Pull request merged**